### PR TITLE
Link to version of docs corresponding to an npm tag

### PR DESCRIPTION
**Pull request by maier49**
This allows a an npm tag to be configured (and defaults it to `beta2` for the applicable packages) when generating API docs. The version corresponding to that tag is resolved and if it exists that is used as the version to link to from the index instead of the latest.

It also adds a new test case and fixes an issue with the existing test case on Windows.

### CODE REVIEW

File: support/commands/getApiPackages.ts [hunk 1]
```diff
@@ -9,25 +11,49 @@ export interface DocumentedApi {
...
+}
+
 /**
  * Provides information on installed API packages
  * @param apiDirectory the directory where packages reside
+ * @param npmTags Optional NPM tags to use to determine the appropriate version to display
  */
-export default function getApiPackages(apiDirectory: string): DocumentedApi[] {
-	return readdirSync(apiDirectory).filter(function (file: string) {
+export default function getApiPackages(apiDirectory: string, npmTags: { [ dir: string ]: NpmTag } = {}): Promise<DocumentedApi[]> {
```
Comment (agubler) [inline support/commands/getApiPackages.ts]: might be clearer using asyc/await?

**Comment by maier49** [comment]
Since `beta3` points at a version that the current logic in another part of the build considers "old" (0.1.0) it gets filtered out before it ever gets to the logic added by this PR. The changes in the PR still work,  but we'll need additional changes to get the desired behavior.

**Comment by kitsonk** [comment]
Putting this on hold because we are removing the API docs from the website for now.

**Comment by dylans** [comment]
Given that we're dropping the API doc viewer at this time, I'm closing this one.

**Pull request closed without merging**